lib/Doctrine/Common/Annotations/CachedReader.php

```diff
@@ -204,34 +224,34 @@ private function fetchFromCache($cacheKey, ReflectionClass $class)
      */
     private function saveToCache($cacheKey, $value)
     {
-        $this->cache->save($cacheKey, $value);
+        $cacheKey = strtr($cacheKey, '\\', '.');
+        $this->cache->set($cacheKey, $value);
         if ($this->debug) {
-            $this->cache->save('[C]'.$cacheKey, time());
+            $this->cache->set('[C]'.$cacheKey, time());
         }
     }
 
     /**
      * Checks if the cache is fresh.
      *
-     * @param string           $cacheKey
-     * @param ReflectionClass $class
+     * @param string $cacheKey should already be sanitized to be compatible
+     *                         with PSR
      *
      * @return boolean
      */
     private function isCacheFresh($cacheKey, ReflectionClass $class)
     {
-        if (null === $lastModification = $this->getLastModification($class)) {
+        if (false === $lastModification = $this->getLastModification($class)) {
             return true;
         }
 
-        return $this->cache->fetch('[C]'.$cacheKey) >= $lastModification;
+        return $this->cache->get('[C]'.$cacheKey) >= $lastModification;
     }
 
     /**
      * Returns the time the class was last modified, testing traits and parents
      *
-     * @param ReflectionClass $class
-     * @return int
+     * @return int|false
```

Comment:
wrong phpdoc. `max` return `false` only for empty arrays, and we always pass a non-empty array.